src/app/views/river-detail/components/main-tab/components/beta-box/components/beta-box-edit-modal.vue

```diff
@@ -48,6 +48,19 @@
         :min="0"
         :mobile="windowWidth <= $options.breakpoints.md"
       />
+      <cv-text-input
+        v-model="formData.permiturl"
+        class="mb-spacing-md"
+        label="Permit URL"
+        :mobile="windowWidth <= $options.breakpoints.md"
+      />
+      <cv-text-input
+        v-model="formData.permitinfo"
+        class="mb-spacing-md"
+        label="Permit Information"
+
+        :mobile="windowWidth <= $options.breakpoints.md"
```

Comment:
does this do anything? I'm trying to look at the carbon docs and code and not seeing it, but as we have all discussed many times, those docs are a nightmare, so maybe I'm missing it